src/oscar/static_src/oscar/scss/page/layout.scss

```diff
@@ -0,0 +1,33 @@
+// OSCAR PAGE STRUCTURE
+// --------------------
+
+body {
+  background-color: $gray-200;
+}
+
+.container-fluid {
+  // Here is the page width: editable in page/mixins.less
+  max-width: $oscar-page-width;
+}
+
+.row {
+  box-sizing: border-box;
+}
+
+// The page container
+.page .page_inner {
+  background: $white;
+  -webkit-border-radius: 0 0 6px 6px;
```

Comment:
`box-shadow` and `border-radius` are now widely supported - we shouldn't need the browser prefixes.